tests/api_workflow/test_api_workflow_client.py

```diff
@@ -79,3 +80,8 @@ def raise_connection_error(*args, **kwargs):
                     file=mock.Mock(),
                     signed_write_url='',
                 )
+
+def test_user_agent_header() -> None:
+    client = ApiWorkflowClient(token="")
```

Comment:
Is this mocked now?

Reply:
No, this test is completely unmocked.

Reply:
Ok, I just saw that it works anyways 👍 

Can you please change the test as follows:
```python

 # patch lightly.__version__, platform.system, platform.release, platform.platform, platform.processor and platform.python_version here

client = ApiWorkflowClient(token"")
assert client.api_client.user_agent == "Lightly/mock_version (mock_system/mock_release; mock_platform; mock_processor;) python/mock_python_version"

Reply:
I mocked everything.